content/07.pathogenesis.md

```diff
@@ -6,11 +6,24 @@ This section would also be great for the introduction of zoonotic diseases which
 
 #### Primary Transmission and Viral Entry
 
+Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052]. 
 ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145; @doi:10.1007/s00134-020-05985-9; @doi:10.1128/JVI.00127-20]. 
```

Comment:
I think it would be more succinct if we just combined this sentence with the first sentence, and add 2 references. Perhaps like this: 

Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].